packages/atlas-service/src/main.ts

```diff
@@ -186,12 +196,12 @@ export class AtlasService {
 
   private static setupPlugin(serializedState?: string) {
     this.plugin = this.createMongoDBOIDCPlugin({
-      redirectServerRequestHandler(data) {
+      redirectServerRequestHandler: (data) => {
         if (data.result === 'redirecting') {
           const { res, status, location } = data;
           res.statusCode = status;
           const redirectUrl = new URL(
-            'https://account.mongodb.com/account/login'
+            `${this.authPortalBaseUrl}/account/login`
```

Comment:
Very optional suggestion, should this just be

```suggestion
            `${this.authPortalBaseUrl}`
```

with `/account/login` as part of the env var?